test/runtests.jl

```diff
@@ -1,8 +1,15 @@
 module TestRematch
 
+using Compat.Test
+using Compat.InteractiveUtils
 using Rematch
 import Rematch: MatchFailure
-using Compat.Test
+
+if VERSION < v"0.7.0-DEV"
+    AbstractRange = Range
+end
```

Comment:
Ah, sorry i wasn't super clear. I think after `using Compat.InteractiveUtils` you don't need this `AbstractRange = Range` assignment at all. you get the for free from `Compat.InteractiveUtils`.

Basically, `Compat` contains all these `VERSION` checks so you don't need to liter your code with them. So the version of `InteractiveUtils` in `Compat` contains `AbstractRange`, and in `VERSION < v"0.7.0-DEV"` it's simply aliased to `Range`.